# -*- coding: utf-8 -*-
"""Task 3.ipynb

Automatically generated by Colaboratory.

Original file is located at
    https://colab.research.google.com/drive/1XikClTMTNaGH9gHLO6S3eFNgl1vQVx5W

Task 3
"""

import logging

name = 'John'
logging.error(f'{name} raised an error')

import logging
a = 5
b = 0
try:
 c = a / b
except Exception as e:
 logging.error("Exception occurred", exc_info=True)

import logging
a = 5
b = 0
try:
 c = a / b
except Exception as e:
 logging.exception("Exception occurred")